src/main/java/no/nav/familie/inntektsmelding/forvaltning/FpDokgenRestTjeneste.java

```diff
@@ -130,9 +148,13 @@ public Response genererPdf(@Valid @NotNull InntektsmeldingRequest inntektsmeldin
             .toList();
     }
 
-    private List<RefusjonsendringEntitet> mapRefusjonsendringer(List<EndringRefusjonDto> refusjonsendringer) {
-        return refusjonsendringer.stream().
-            map(periode -> new RefusjonsendringEntitet(periode.fom(), periode.beloep()))
+    private List<RefusjonsendringEntitet> mapRefusjonsendringer(LocalDate startdato, LocalDate opphørsdato, List<Refusjon> refusjonsendring) {
```

Comment:
Hmm, hvorfor endres denne nå? Var den feil før?

Reply:
denne var feil (ikke oppdatert til siste endringer)- det er endepunktet som jeg brukte for å trigge genering av inntektsmeldingspdf - ikke i bruk ellers. Men brukte den igjen for å trigge ny mal